hasura.planx.uk/migrations/1738840144554_add_flow_name_to_feedback_summary/down.sql

```diff
@@ -0,0 +1,32 @@
+DROP VIEW "public"."feedback_summary";
```

Comment:
Not sure if this is something I've noticed before - I had to `DROP` this as I was changing the structure. This meant I had to re-set up relationships and permissions in Hasura - kind of annoying and easy to miss. Possibly the cause of the "broken access controls" issue here maybe? Did we set this up correctly and test it once, then on update of a view not reconfigure it properly?